### ISSUE TITLE
icalendar events with RECURRENCE-ID are displayed twice

### ISSUE THREAD

**Issue by christoph-kluge** [fullcalendar/fullcalendar#6451]
### Reduced Test Case

[Test Case](https://codesandbox.io/s/setprop-typing-forked-d8jky)

### Bug Description

The icalendar plugin (#5969) does not support RECURRENCE-ID.  If I create/use an .ics file with a proper implementation of RECURRENCE-ID then it will display the event twice. The original event and the changed event.

### Screenshots

<img width="1298" alt="Screenshot 2021-07-12 at 20 19 51" src="https://user-images.githubusercontent.com/1446269/125337023-c2cb7500-e34e-11eb-8cde-650c618ea5f8.png">

### Sample VEVENT 

The main VEVENT object
```
BEGIN:VEVENT
UID:af80d1e9-2adb-4e06-8483-3130c18305b1
DTSTAMP:20210712T175548Z
CLASS:PUBLIC
CREATED:20210712T150845Z
DTSTART;TZID=Europe/Berlin:20210705T140000
DTEND;TZID=Europe/Berlin:20210705T143000
LAST-MODIFIED:20210712T150845Z
ORGANIZER;CN=Christoph Kluge:MAILTO:work@christoph-kluge.eu
RRULE:FREQ=DAILY
SEQUENCE:0
SUMMARY:Daily Recurring
TRANSP:OPAQUE
END:VEVENT
```

The child VEVENT object
```
BEGIN:VEVENT
UID:af80d1e9-2adb-4e06-8483-3130c18305b1
DTSTAMP:20210712T175548Z
CLASS:PUBLIC
CREATED:20210712T150845Z
DTSTART;TZID=Europe/Berlin:20210708T141500
DTEND;TZID=Europe/Berlin:20210708T144500
LAST-MODIFIED:20210712T150845Z
RECURRENCE-ID;TZID=Europe/Berlin:20210708T140000
SEQUENCE:1
SUMMARY:Daily Recurring (Update-1)
TRANSP:OPAQUE
END:VEVENT
```

**Comment by acerix** [comment]
Would you be able to post a [runnable, stripped-down demonstration of the bug](http://fullcalendar.io/wiki/Reduced-Test-Cases/)? Would really appreciate it because the time saved reproducing will be time spent fixing.

**Comment by christoph-kluge** [comment]
@acerix I am not able to create a codepen or jsfiddle example because I can't add .ics files w/o having a account or a PRO subscription. 

I can add a full reproducible .ics file if this helps?

**Comment by christoph-kluge** [comment]
@acerix addded it into the description. Thanks to github gists. 

[example on codesandbox.io](https://codesandbox.io/s/setprop-typing-forked-d8jky)

**Comment by seanmichaelragan** [comment]
@christoph-kluge @acerix Having exactly this issue. May I ask why it was closed? Thank you.

**Comment by christoph-kluge** [comment]
@seanmichaelragan it was closed because there was no reproducible example before I have edited the original post. Now we need to wait for @acerix to reopen it. 

Actually I'm not sure If I should file a new issue or wait until someone reopen's it. Dunno how the maintainers handle these cases. Since this issue is very fresh I am waiting for now.

**Comment by acerix** [comment]
Thanks for the repro, it looks like a bug so I'll reopen.

**Comment by arshaw** [comment]
released in [5.9.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v5.9.0)


you'll see there's only one even per day in the updated repro:
https://codesandbox.io/s/setprop-typing-forked-4ihnc?file=/src/calendar.tsx
